pydy/viz/scene.py

```diff
@@ -460,9 +595,37 @@ def remove_static_html(self, force=False):
         else:
             print("aborted!")
 
+    def copy_resources(self, silent=False):
+        """
+        Copies simulation and static files to a directory pointed
+        by self.static_url attribute.
+        It overwrites if such directory is already present.
+        """
+        dst = self.static_url
+        if os.path.exists(dst):
+            if not silent:
+                print("Cleaning previous static data")
+            distutils.dir_util.remove_tree(dst)
+
+        src = os.path.join(os.path.dirname(__file__), 'static')
+
+        if not silent:
+            print("Copying static data.")
+        distutils.dir_util.copy_tree(src, dst)
+
+        if not silent:
+            print("Copying Simulation data.")
+        try:
+            shutil.move(os.path.join(os.getcwd(), self._scene_json_file),
+                        os.path.join(self.static_url, self._scene_json_file))
+            shutil.move(os.path.join(os.getcwd(), self._simulation_json_file),
+                        os.path.join(self.static_url, self._simulation_json_file))
+        except IOError:
+            pass
```

Comment:
No, but right now I have an issue with ipython notebook. When we run notebook seconds time without reloading, no visualization file is created and this needs to be fixed. This will be fixed before merging.